otherlibs/dune-rpc/private/types.ml

```diff
@@ -27,7 +27,7 @@ end
 module Version = struct
   type t = int * int
 
-  let latest = 3, 13
+  let latest = 3, 14
```

Comment:
Note I had to change this to be able to run the test. Not sure if bumping this version involves more work / should be done in a different PR.

Reply:
It's a bit surprising that this version is stored in `dune-rpc` library 🤔